### PR TITLE
Add separate forms for SE Apartment Rent / BRF

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add separate forms for SE Apartment Rent & BRF

Remove ownership question from both new forms and add to initial data

Update copy to match Figma

## Justify why they are needed

Support separate rent and BRF PDPs

I kept `SE_APARTMENT` form since we use it on some test pages

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1876]: https://hedvig.atlassian.net/browse/GRW-1876?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #985** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/985" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/985?utm_source=stack-comment).

File: apps/store/src/services/PriceCalculator/data/SE_APARTMENT_RENT.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { Template } from '../PriceCalculator.types'
+
+export const SE_APARTMENT_RENT: Template = {
+  name: 'SE_APARTMENT_RENT',
+  initialData: { subType: 'RENT' },
```
Comment (gustaveen) [inline apps/store/src/services/PriceCalculator/data/SE_APARTMENT_RENT.ts:5]: Ah nice! Was wondering how we set predefined data in the forms 👌

**Review by gustaveen — APPROVED** [review]
Fantastic! 🎉

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/985?utm_source=stack-comment).


Job Reference: NyqCkgg7tzcsRE7hLfXP

**Pull request merged**